modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,186 @@
-= Section 2
+= Elyra Pipelines
 
-This is _Section 2_ of _Chapter 1_ in the *hello* quick course....
\ No newline at end of file
+Elyra provides a visual pipeline editor for building pipelines from Python and R scripts as well as Jupyter notebooks, simplifying the conversion of multiple files into batch jobs or workflows. A pipeline in Elyra consists of nodes that are connected with each other to define execution dependencies.
+
+image::elyra_jupyter.png[Jupyter with elyra]
+
+Elyra's visual pipeline editor lets you assemble pipelines by dragging and dropping supported files onto the canvas and defining their dependencies. After you've assembled the pipeline and are ready to run it, the editor takes care of generating the Tekton YAML definition on the fly and submitting it to the Data Science Pipelines backend.
+
+== Creating a Data Science Pipeline with Elyra
+
+In order to create Elyra pipelines with the visual pipeline editor:
+
+* Launch JupyterLab with the Elyra extension installed.
+* Create a new pipeline by clicking on the Elyra `Pipeline Editor` icon.
+* Add each node to the pipeline by dragging and dropping notebooks or scripts from the file browser onto the pipeline editor canvas.
```

Comment:
This might be a nice place to add the recommendation about using scripts rather than notebooks.
Something like this maybe (feel free to adjust):

```suggestion
* Add each node to the pipeline by dragging and dropping notebooks or scripts from the file browser onto the pipeline editor canvas.
+
[TIP]
====
Although Elyra supports both notebooks and scripts, the use of scripts is recommended.
====
```

Reply:
Yes, good point. I've added a note in the pipeline creation section.